### ISSUE TITLE
truss push faulty on windows?

### ISSUE THREAD

**Issue by TidorP** [basetenlabs/truss-examples#301]
ERROR (<class 'NotImplementedError'>: cannot instantiate 'PosixPath' on your system

^ this is probably because truss expects posix systems like macos, linux. Needs WindowsPath.

**Comment by TidorP** [comment]
@maxIrvine no, so what I ended up doing is deploying through a macOS (I had a macbook with me too). Then the request can be done through any machine including (cuz it's just a request to the deployed model). But yeah unfortunately it seems you need an ubuntu or mac machine to push the model. Afterwards, you can access it anywhere.

**Comment by squidarth** [comment]
Hey, sorry folks, will look into this
